mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -47,6 +47,7 @@ private InputFormatConfig() {
   public static final String CATALOG = "iceberg.mr.catalog";
   public static final String HADOOP_CATALOG_WAREHOUSE_LOCATION = "iceberg.mr.catalog.hadoop.warehouse.location";
   public static final String CATALOG_LOADER_CLASS = "iceberg.mr.catalog.loader.class";
+  public static final String COLUMN_PROJECTIONS = "iceberg.mr.column.projections";
```

Comment:
The wording we've generally settled on is that `project` is used to set a requested schema and `select` is used to select columns like a `SELECT` statement in SQL. Following that convention, when we pass a schema we call it a "projection" and when we pass columns we call them "selected columns".

I think it would be good to make this more clear, since this is passing columns to project and not a schema. How about `SELECTED_COLUMNS = "iceberg.mr.selected.columns"`?